guides/doc-Planning_for_Project/master.adoc

```diff
@@ -33,6 +33,8 @@ include::common/assembly_common-deployment-scenarios.adoc[leveloffset=+1]
 
 include::common/assembly_provisioning-requirements.adoc[leveloffset=+1]
 
+include::common/modules/con_deploying-foreman-on-aws.adoc[leveloffset=+1]
```

Comment:
This looks strange at the top level. Since the content is only 2 sentences, I would just move them to con_scenarios-for-foreman-on-aws.adoc and delete this module.